### PR TITLE
BZ1936838: Adds localvolumeset and localvolumediscovery

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1936838

OCP Version: 4.6+

Direct Doc Preview Link: https://deploy-preview-42142--osdocs.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-local.html#local-storage-uninstall_persistent-storage-local

@duanwei33 PTAL and let me know your feedback.

### CODE REVIEW

**Comment by duanwei33** [comment]
It looks good to me.
@chao007 Could you take a look as well?

**Comment by chao007** [comment]
@darshan-nagaraj It is better to add "no bound/released PVs found for localvolume or localvolumeset before delete them".

File: modules/persistent-storage-local-uninstall-operator.adoc [hunk 1]
```diff
@@ -20,11 +20,13 @@ there might be indeterminate behavior if the Operator is uninstalled and reinsta
 
 .Procedure
 
-. Delete any local volume resources in the project:
+. Delete any local volume resources installed in the project such as `localvolume`, `localvolumeset`, and `localvolumediscovery`:
```
[the remark this answers is not in the document]
Comment (darshan-nagaraj) [reply modules/persistent-storage-local-uninstall-operator.adoc]: Done!

**Pull request merged**